osu.Game.Rulesets.Catch/Mods/CatchModDifficultyAdjust.cs

```diff
@@ -10,7 +10,7 @@ namespace osu.Game.Rulesets.Catch.Mods
 {
     public class CatchModDifficultyAdjust : ModDifficultyAdjust
     {
-        [SettingSource("Fruit Size", "Override a beatmap's set CS.")]
+        [SettingSource("Circle Size", "Override a beatmap's set CS.", 0)]
```

Comment:
I don't think this is a proper change. We've already fixed up `Circle SIze` -> `Key Amount` for mania in song select (albeit with a hack of sorts), so if we've gone that way then we should probably also fix up `Fruit Size` there as well instead of changing the mod this time.

Reply:
peppy says [here](https://discordapp.com/channels/188630481301012481/188630652340404224/676283363430236161) that all naming should be consistent with web. Do I change it back anyway?

Reply:
Ah, fair call then, missed that. Disregard for now.